steel/testdata/rpc_cache.json

```diff
@@ -475,6 +475,76 @@
         ]
       }
     ],
+    [
+      {
+        "block_no": 19493153,
+        "address": "0xdac17f958d2ee523a2206206994597c13d831ec7",
```

Comment:
The same `storage_proof` is already in the cache, only with one index missing. This old proof should be removed to keep the cache as small as possible

Reply:
Is there a clean way to prune the old data or does this have to be done manually or by regenerating the cache?

Reply:
Wait, what is the duplicate? I didn't remove any other test, so the query with one less value is also needed for the old test. Am I overlooking one here/did I remove the wrong one? Seems the queries have to match exactly.

Reply:
Oh, sorry, you are absolutely right!
I just created the test cache from scratch, and it came up with a file where several lines were deleted.  However, upon closer inspection, it also adds the same number of lines in a different location. I am a bit confused though, I was expecting the cache file to be deterministic 🤔